.env

```diff
@@ -1 +1,2 @@
-URI=mongodb+srv://<USER-HERE>:<PASSWORD-HERE>@cluster0.aggji.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
\ No newline at end of file
+USER=trove
```

Comment:
Are these two lines (1 & 2) old code that can be removed? That's the only red flag I see, everything else looks good.

Reply:
Yeah i suppose if each of us has these vars on our own machines it we won't need them committed to the repo

Reply:
Oh I'm silly, I didn't even realize these were in the .env file. Yeah if you could write something like [Insert username here], [insert pw here] as placeholders then the .env would not have to be gitignored. Does that make sense? If not let me know what you were thinking.